examples/demos/boot_from_san_and_migration/README.md

```diff
@@ -19,8 +18,8 @@ This playbook exemplifies how to create a server profile using a server profile
 
 ### [Step 4 - Migrate a data volume to another server profile](step_4_migrate_data_volume.yml)
 This playbook exemplifies how to migrate a data volume from one server profile to another. It performs 2 tasks:
-1. Task copied from step 2 when the extra volume was attached. This task updates the server profile 'profile_san' removing data volume identified by id 2 (returning to the original state - same as in the step 1).
-2. Task copied from step 3 to update server profile 'profile_i3s' inserting data volume removed from previous task. The connectionId used by the storage paths need to be updated to match with the SAN connections. In this case, the SAN connection ids in the server profile template were 4 and 5. 
+1. Updates server profile 'profile_san' to match its original state on step 1, removing data volume identified by id 2.
```

Comment:
Updates server profile 'profile_san' to match its original state on step 1, removing **the** data volume identified by id 2.

Reply:
Done